### ISSUE TITLE
AML: Alias opcode support required

### ISSUE THREAD

**Issue by rw-vanc** [rust-osdev/acpi#136]
On HP-dq3xxx, Alias opcode 0x06 is used, causing the parser to report an error.

**Comment by rw-vanc** [comment]
I have an implementation that accepts Alias, but I have not written the `as_integer` etc. functionality. Let me know if you want this.

**Comment by IsaacWoods** [comment]
Yep, we don't support aliases yet! Feel free to open a PR with your current impl - I always imagined they should be handled in the namespace layer (perhaps with a separate map, but not sure), so not sure this should need functionality in e.g. `as_integer`, but happy to review!
